identity/aziot-identityd-config/src/lib.rs

```diff
@@ -57,7 +57,7 @@ pub struct Settings {
 
 impl Settings {
     pub fn default_cloud_timeout() -> u64 {
-        10
+        60
```

Comment:
We might want it to be a bit higher since 60s is the request timeout on the server side, and the client does extra work of DNS lookup, establishing the TCP connection and sending the request that we include in the request timeout on our end.

70s should be good enough.

Reply:
In testing the server never took longer than 45 seconds, so the 60s was already a small buffer

Reply:
Yes, that's what Viktor observed as well. But the 60s timeout is what they document, so we should take them at their word.